config/localization/en/app.json

```diff
@@ -1968,6 +1969,7 @@
     "BROADCAST_ERROR_2001": "Reduce-only orders cannot increase the position size",
     "BROADCAST_ERROR_2002": "Reduce-only orders cannot change the position side",
     "BROADCAST_ERROR_2003": "Post-only order would cross one or more maker orders",
+    "BROADCAST_ERROR_2005": "Order would violate isolated subaccount constraints.",
```

Comment:
nit: seems like periods aren't used in other broadcast errors, consider removing?

Reply:
Will do! 🙏